src/app.js

```diff
@@ -8,7 +8,7 @@ const homeUrl = process.env.TOP_LEVEL;
 const App = () => {
   return (
       <Location homeUrl={homeUrl}>
-        <ExplorerUI homeUrl={homeUrl}/>
+        <ExplorerUI />
```

Comment:
This is a small change but seems like a big improvement, pulling that prop out of the explorer